runelite-client/src/main/java/net/runelite/client/plugins/rightclick/RightclickOptionsConfig.java

```diff
@@ -0,0 +1,170 @@
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.rightclick;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+import net.runelite.client.plugins.rightclick.config.MenuHighlightMode;
+
+import java.awt.*;
+
+@ConfigGroup("rightclick")
+public interface RightclickOptionsConfig extends Config
+{
+	@ConfigItem(
+		keyName = "highlightedRightclickItems",
+		name = "Highlighted Menu Items",
+		description = "List of right-click menu entries to highlight, separated by commas.",
+		position = 0
+	)
+	default String getHighlightedRightclickItems()
+	{
+		return "";
+	}
+
+	@ConfigItem(
+		keyName = "dimmedItems",
+		name = "Dimmed Items",
+		description = "List of right-click menu entries to dim, separated by commas",
+		position = 1
+	)
+	default String getDimmedItems()
+	{
+		return "";
+	}
+
+	@ConfigItem(
+		keyName = "menuHighlightMode",
+		name = "Menu Highlight Mode",
+		description = "Configures what to highlight in right-click menu",
+		position = 2
+	)
+	default MenuHighlightMode menuHighlightMode()
+	{
+		return MenuHighlightMode.NAME;
+	}
+
+	@ConfigItem(
+		keyName = "highlightedColor",
+		name = "Highlighted items color",
+		description = "Configures the color for highlighted entries",
+		position = 3
+	)
+	default Color highlightedColor()
+	{
+		return Color.decode("#FF66AA");
+	}
+
+	@ConfigItem(
+		keyName = "dimmedColor",
+		name = "Dimmed items color",
+		description = "Configures the color for dimmed entries",
+		position = 4
+	)
+	default Color dimmedColor()
+	{
+		return Color.decode("#808080");
+	}
+
+	@ConfigItem(
+		keyName = "hideExamine",
+		name = "Hide \"Examine\"",
+		description = "Hides Examine from all right-click menus",
+		position = 5
+	)
+	default boolean hideExamine()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "hideTradeWith",
+		name = "Hide \"Trade with\"",
+		description = "Hides Trade from all right-click menus",
+		position = 6
+	)
+	default boolean hideTradeWith()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "hideFollow",
+		name = "Hide \"Follow\"",
+		description = "Hides Follow from all right-click menus",
+		position = 7
+	)
+	default boolean hideFollow()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "hideWithdraw5",
+		name = "Hide \"Withdraw-5\"",
+		description = "Hides Withdraw-5 from all right-click menus",
+		position = 8
+	)
+	default boolean hideWithdraw5()
```

Comment:
This is just a way around not swapping withdraw-x

Reply:
It makes it the second option.

Reply:
I would like to not touch the bank menu at all.